src/invidious/comments/youtube.cr

```diff
@@ -303,6 +303,7 @@ module Invidious::Comments
     if format == "html"
       response = JSON.parse(response)
       content_html = Frontend::Comments.template_youtube(response, locale, thin_mode)
+      content_html = Comments.replace_external_links(content_html)
```

Comment:
Can you explain why you added this here? Is it because you missed one link to replace, [as mentioned here?](https://github.com/iv-org/invidious/pull/4826#discussion_r1712965531)